src/main/webapp/app/text-editor/text-editor.component.ts

```diff
@@ -163,27 +177,24 @@ export class TextEditorComponent implements OnInit {
         }
         this.submission.text = this.answer;
         this.submission.language = this.textService.predictLanguage(this.submission.text);
-        const confirmSubmit = window.confirm(this.submissionConfirmationText);
 
-        if (confirmSubmit) {
-            this.submission.submitted = true;
-            this.textSubmissionService.update(this.submission, this.textExercise.id).subscribe(
-                response => {
-                    this.submission = response.body!;
-                    this.result = this.submission.result;
+        this.submission.submitted = true;
+        this.textSubmissionService.update(this.submission, this.textExercise.id).subscribe(
+            response => {
+                this.submission = response.body!;
+                this.result = this.submission.result;
 
-                    if (this.isActive) {
-                        this.jhiAlertService.success('artemisApp.textExercise.submitSuccessful');
-                    } else {
-                        this.jhiAlertService.warning('artemisApp.textExercise.submitDeadlineMissed');
-                    }
-                },
-                err => {
-                    this.jhiAlertService.error('artemisApp.modelingEditor.error');
-                    this.submission.submitted = false;
-                },
-            );
-        }
+                if (!this.isAlwaysActive) {
```

Comment:
For me it showed warning alert, when submitting in time

Reply:
Should be fixed now